### PR TITLE
Load more results for PropertyLookup on scroll

### PR DESCRIPTION
**Pull request by bereket-WMDE**
Bug: T268508

### CODE REVIEW

File: src/components/PropertyLookup.vue [hunk 1]
```diff
@@ -7,6 +7,7 @@
 		:search-input.sync="search"
 		:placeholder="$i18n('query-builder-property-lookup-placeholder')"
 		:label="$i18n('query-builder-property-lookup-label')"
+		v-on:scroll="handelScroll"
```
Comment (Ladsgroup) [inline src/components/PropertyLookup.vue]: typo: handle :D

File: tests/unit/data-access/FetchSearchEntityRepository.spec.ts [hunk 2]
```diff
@@ -66,6 +66,7 @@ describe( 'FetchSearchEntityRepository', () => {
 			origin: '*',
 			limit,
 			offset,
```
Comment (Ladsgroup) [inline tests/unit/data-access/FetchSearchEntityRepository.spec.ts]: The offset here is not needed at all: https://www.wikidata.org/w/api.php?action=help&modules=wbsearchentities

File: src/data-access/FetchSearchEntityRepository.ts [hunk 3]
```diff
@@ -28,6 +28,7 @@ export default class FetchSearchEntityRepository implements SearchEntityReposito
 		}
 		if ( offset ) {
 			params.offset = `${offset}`;
```
Comment (Ladsgroup) [inline src/data-access/FetchSearchEntityRepository.ts]: this should be dropped

**Review by Ladsgroup — CHANGES_REQUESTED** [review]
Thanks for doing it!

File: src/components/PropertyLookup.vue (hunk 1 above)
Comment (bereket-WMDE) [reply src/components/PropertyLookup.vue]: oops

File: tests/unit/data-access/FetchSearchEntityRepository.spec.ts (hunk 2 above)
Comment (bereket-WMDE) [reply tests/unit/data-access/FetchSearchEntityRepository.spec.ts]: good catch

File: src/data-access/FetchSearchEntityRepository.ts (hunk 3 above)
Comment (bereket-WMDE) [reply src/data-access/FetchSearchEntityRepository.ts]: removed

**Review by Ladsgroup — APPROVED** [review]
LGTM now, thanks!

**Pull request merged**